robottelo/decorators/__init__.py

```diff
@@ -368,22 +406,29 @@ def bz_bug_is_open(bug_id):
     :rtype: bool
 
     """
-    bug = None
     try:
         bug = _get_bugzilla_bug(bug_id)
     except BugFetchError as err:
         LOGGER.warning(err)
         return False
-    # NOT_FOUND, ON_QA, VERIFIED, RELEASE_PENDING, CLOSED
-    if bug is None or bug.status not in BZ_OPEN_STATUSES:
+    else:
+        # NEW, ASSIGNED, MODIFIED, POST
+        if bug.status in BZ_OPEN_STATUSES:
+            return True
+
```

Comment:
I can have bug OPEN for 6.2 and 6.3 - no flags here ? I know it is limited workflow
It was said it will never happen (but today it happened).
If it happens here we are required to clone anyway ...